.mergify.yml

```diff
@@ -101,6 +101,17 @@ pull_request_rules:
         - files~=^\.mergify\.yml$
     actions:
       delete_head_branch:
+  - name: notify the backport has not been merged yet
+    conditions:
+      - -merged
+      - -closed
+      - author=mergify[bot]
+      - "#check-success>0"
```

Comment:
Does this mean "number of successful checks is greater than zero"? will this include PRs with some failing check? (I guess so, just confirming just in case :)).

Reply:
That's a possibility. I used the checks to avoid immediate events, but if we use your suggestion of ` backport is older than some days?` then we can remove it

Reply:
I am ok with this check, and actually seeing that this helps to avoid immediate events, I see that the check for older PRs would be even less needed.